test/db/cmd/cmd_wox

```diff
@@ -72,7 +72,7 @@ wox 0xabcdef010203@!10
 ps 14
 EOF
 EXPECT=<<EOF
-helloworld\x00\x00\x00\x00
+helloworld
```

Comment:
ps 18 should show the zeroes. can you also testt this, as well as psz as i said in my previous comment?. also this is not testing the charsets at all

Reply:
In my opinion ps should not have a different behavior if the space is specified. If we want 0s we should use psz.


This pull request is not for test the charset. I will test the charset in another pull request.

Reply:
Exactly. i agree that `ps` and `ps $b` shuold print the same. but your PR is making ps do the same thing as psz. and this is not correct. Can you please make `ps` just take the blocksize as length instead of behaving like psz?

Also, can you make all subcommands of `ps` that are not handled to print an error and do nothing instead of doing something random?

Reply:
![Screenshot 2020-12-28 at 18 42 53](https://user-images.githubusercontent.com/6431515/103233121-84371580-493c-11eb-83ce-11b51cd005d1.png)

Reply:
Ok I understand. I did not know the command psz before. I doubt. I think reverse the effects of ps and psz might be more intuitive. Not sure.

`ps $b` prints the block size. So it is different of course. Why should it be same?